woocommerce/woocommerce-ios
# Track `signed_in` event during site credentials login

## Description
This PR starts tracking the `signed_in` tracks event when the client handles the site credential login. 


## Testing instructions

Prerequiste - Create a JN site without Jetpack and with WooCommerce

- Install and launch the app 
- Tap "Enter Your Store Address" and enter the JN site
- Tap Continue and enter your wp-admin credentials in the next screen
- You should land in the dashboard screen
- From the Xcode console, ensure that the following events are tracked in the same order
```
Tracked application_passwords_new_password_created, properties: [AnyHashable("scenario"): "generation"]
Tracked signed_in
```

## Screenshots
NA

---
